common/src/OpenSpaceNet.cpp

```diff
@@ -459,31 +463,17 @@ void OpenSpaceNet::processSerial()
     skipLine();
     OSN_LOG(info) << "Reading image...";
 
-    unique_ptr<boost::progress_display> openProgress;
     if(!args_.quiet) {
-        openProgress = make_unique<boost::progress_display>(50);
+        pd_->start();
     }
 
-
     auto startTime = high_resolution_clock::now();
-    auto mat = GeoImage::readImage(*image_, bbox_, regionFilter_.get(), [&openProgress](float progress) -> bool {
-        size_t curProgress = (size_t)roundf(progress*50);
-        if(openProgress && openProgress->count() < curProgress) {
-            *openProgress += curProgress - openProgress->count();
-        }
-        return true;
+    auto mat = GeoImage::readImage(*image_, bbox_, regionFilter_.get(), [this](float progress) -> bool {
+        pd_->update(0, progress);
+        return !pd_->cancelled();
     });
 
     duration<double> duration = high_resolution_clock::now() - startTime;
-    OSN_LOG(info) << "Reading time " << duration.count() << " s";
-
-    skipLine();
-    OSN_LOG(info) << "Detecting features...";
```

Comment:
After discussion the output has been formatted to look like this:
`Initializing the output feature set...

Processing...

|  Reading  | Detecting |
+-----------+-----------+
|  100.0%   |  100.0%   |

Reading time 0.144755 s
Detection time 16.3113 s


Performing category filtering...

Performing non-maximum suppression...
93 features detected.
Saving feature set...`